File: String-1/src/makeTags/MakeTags.java
package makeTags;

public class MakeTags {
    //Dada a "tag" e a "word" então retorne a String como <"tag">"word"</"tag">.

    public String makeTags(String tag, String word) {
        return "<"+tag+">"+word+"</"+tag+">";
    }
}

File: Python/String-1/make_tags.py
"""
Dada 2 Strings ("tag" e "word"), retorne uma outra seguindo a seguinte lógica:
"<'tag'>'word'</'tag'>"
"""


def make_tags(tag, word):
    return "<{}>{}</{}>".format(tag, word, tag)


print(make_tags("i", "hello"))
